consensus/src/chained_bft/block_storage/pending_votes.rs

```diff
@@ -105,7 +105,7 @@ impl PendingVotes {
             Err(VerifyError::TooLittleVotingPower { voting_power, .. }) => {
                 VoteReceptionResult::VoteAdded(voting_power)
             }
-            _ => panic!("Unexpected verification error, vote = {}", vote),
+            Err(_) => VoteReceptionResult::ErrorAddingVote,
```

Comment:
I actually don't understand why we ever thought it's a good idea to panic in this case. Thanks for fixing this one!

Reply:
Somehow this got blamed on me. I think at that point in time, the context was that we had some really strange error and while we might be able to make some progress, we'd likely be in a wonky state. Arguably we should build better log monitoring so that if we do get in that wonky state, we are alerted and perhaps our deployment framework restarts the process if it is bad enough.